src/User.js

```diff
@@ -10,7 +10,13 @@ class User {
   }
 
   renderUserFirstName() {
-    return this.name;
+    return this.name.split(' ')[0];
+  }
+
+  renderUserFriendNames() {
+    const friendNames = this.friends.filter((friend) => {
+      friend === this.friends.id 
+    })
```

Comment:
We can delete this method since we don't have access to the users.js in this class, we only have access to a single user. Or we could look to see if we could pass in the data if we add in a parameter.